gems/activerecord/6.0.3.2/activerecord.rbs

```diff
@@ -84,6 +84,7 @@ interface _ActiveRecord_Relation[Model, PrimaryKey]
   def find: (PrimaryKey id) -> Model
           | (Array[PrimaryKey]) -> Array[Model]
           | (*PrimaryKey) -> Array[Model]
+          | () { (Model) -> boolish } -> Model
```

Comment:
`find` is nil-able. For example: `User.all.find{false} # => nil`

```suggestion
          | () { (Model) -> boolish } -> Model?
```

Reply:
Ah. Right... 🙇‍♂️

Reply:
Ah, thank you!